network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreIT.java

```diff
@@ -20,57 +22,67 @@
 import com.powsybl.network.store.server.CassandraConfig;
-import com.powsybl.network.store.server.CassandraConstants;
 import com.powsybl.network.store.server.NetworkStoreApplication;
 import com.powsybl.sld.iidm.extensions.ConnectablePosition;
 import com.powsybl.ucte.converter.UcteImporter;
 import org.apache.commons.io.FilenameUtils;
-import org.cassandraunit.spring.CassandraDataSet;
-import org.cassandraunit.spring.EmbeddedCassandra;
-import org.cassandraunit.utils.EmbeddedCassandraServerHelper;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.web.server.LocalServerPort;
 import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.TestExecutionListeners;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.*;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import static org.junit.Assert.*;
-import static org.springframework.test.context.TestExecutionListeners.MergeMode.MERGE_WITH_DEFAULTS;
 
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-@ContextConfiguration(classes = {NetworkStoreApplication.class, CassandraConfig.class, NetworkStoreService.class})
-@TestExecutionListeners(listeners = CustomCassandraUnitTestExecutionListener.class,
-                        mergeMode = MERGE_WITH_DEFAULTS)
-@CassandraDataSet(value = "iidm.cql", keyspace = CassandraConstants.KEYSPACE_IIDM)
-@EmbeddedCassandra(timeout = 60000L)
+@ContextConfiguration(classes = {NetworkStoreApplication.class, CassandraConfig.class, NetworkStoreService.class,
+        EmbeddedCassandraFactoryConfig.class, CqlCassandraConnectionFactoryTest.class})
+@EmbeddedCassandra(scripts = {"classpath:create_keyspace.cql", "classpath:iidm.cql"})
 public class NetworkStoreIT {
 
     @LocalServerPort
     private int randomServerPort;
 
+    @Autowired
+    private CqlSessionCassandraConnection cqlSessionCassandraConnection;
+
     private String getBaseUrl() {
         return "http://localhost:" + randomServerPort + "/";
     }
 
-    // This method is provided to avoid timeout when dropping tables
+    private NetworkStoreService createNetworkStoreService() {
+        return new NetworkStoreService(getBaseUrl());
+    }
+
     @Before
-    public void initialize() {
-        EmbeddedCassandraServerHelper.getCluster().getConfiguration().getSocketOptions().setReadTimeoutMillis(60000);
+    public void setup() throws IOException {
+        String truncateScriptPath = getClass().getClassLoader().getResource("truncate.cql").getPath();
```

Comment:
That would be cleaner to not use path for directly getClass().getResourceAsStream to get an input stream on script.

Reply:
Actually you don't need to this by yourself. You can either use `CqlDataSet` or `CqlScript` class.

Reply:
Didn't see this feature. Thanks